src/serializer/utils.js

```diff
@@ -84,6 +84,32 @@ export function getOrDefault<K, V>(map: Map<K, V>, key: K, defaultFn: () => V):
   return value;
 }
 
+export function getObjectPropertyInternalValue(val: ObjectValue, propName: string | SymbolValue): Value | void {
+  let binding;
+  if (typeof propName === "string") {
+    binding = val.properties.get(propName);
+  } else if (propName instanceof SymbolValue) {
+    binding = val.symbols.get(propName);
+  }
+  invariant(binding);
+  let descriptor = binding.descriptor;
+  if (descriptor === undefined) {
+    return;
+  }
+  let value;
+  if (descriptor.value !== undefined) {
+    value = descriptor.value;
+  }
+  if (descriptor.get !== undefined) {
+    value = descriptor.get;
```

Comment:
I think it's a bit confusing the method claims we get a *value* but we could actually get a getter/setter itself. Maybe the method could have been named differently?

Reply:
All three are actually a form of “value”. I don’t need to access the setter though.